src/service/poc_mobile.proto

```diff
@@ -99,13 +140,31 @@ message file_info {
 
 message processed_files { repeated file_info files = 1; }
 
+message coverage_object {
+  bytes pub_key = 1;
+  bytes uuid = 2;
+  string cbsd_id = 3;
+  uint64 coverage_claim_time = 4;
+  repeated radio_hex_signal_level coverage = 5;
+  bool indoor = 6;
+  coverage_object_validity validity = 7;
+}
+
+enum coverage_object_validity {
+  coverage_object_validity_valid = 0;
+  coverage_object_validity_invalid_pub_key = 1;
+  coverage_object_validity_indoor_coverage_is_medium = 2;
+}
```

Comment:
this seems largely redundant with the `coverage_object_req_v1` message type and i don't see the `coverage_object` message used anywhere. it also looks a bit confusing with the `coverage_object` message type and the `coverage_object` field name referring to a UUID in a couple places.

Reply:
i see the resolved comments requesting to change `coverage_object_uuid` to `coverage_object` but it's still confusing seeing that field name in conjunction with a message type of the same name. if `coverage_object` message _is_ still required i would change the `coverage_object` field name to `object_id` or something. i disagree that `coverage_object` implies a reference when there's a message with that name; it implies to me the object message